src/Components/Title/Title.jsx

```diff
@@ -0,0 +1,16 @@
+import style from "./Title.module.css"
+
+export default function Title() {
+    const text = "Die führende SaaS Plattform der Energiewirtschaft für Mid- und Backoffice-Prozesse."
+
+    return (
+        <div className={style.Title}>
+            <h1 className={style.Box}>
+                <h1 className={style.Text}>
+                    {text}
+                </h1>
+            </h1>
+        </div>);
```

Comment:
1. Aufpassen, du hast hier h1 in ein h1 gepackt. Aus dem äußeren h1 bitte ein div machen.
2. Außerdem bitte nochmal prüfen ob du hier wirklich 2 container außen rum benötigst.
3. Eine h1 Überschrift sollte es nur ein mal geben. Benutz am besten h2.
4. Die Komponente sollte noch dynamisch sein -> keine `const text` sondern den text als prop in die komponente übergeben